platformio.ini

```diff
@@ -326,6 +326,12 @@ lib_deps =
 # WLED BUILDS
 # ------------------------------------------------------------------------------
 
+[env:nodemcuv2_160_pir]
```

Comment:
The change to `platformio.ini` is unacceptable. Please revert.

Reply:
Noted , error from my side